client/src/components/ChapterCard.tsx

```diff
@@ -48,30 +53,26 @@ export const ChapterCard: React.FC<ChapterCardProps> = ({ chapter }) => {
             {chapter.description}
           </Text>
         </GridItem>
-        <GridItem colSpan={3}>
-          {chapter.events.map(({ id, name, start_at }) => (
+        <GridItem colSpan={2}>
+          <Heading
+            as="h3"
+            fontSize={'md'}
+            fontWeight={'500'}
+            paddingInline={'1em'}
+            paddingBlock={'.5em'}
+          >
+            Organized Events
+          </Heading>
+          {chapter.events.map(({ id, name }, index) => (
             <Link key={id} href={`/events/${id}`} _hover={{}}>
               <Flex
                 direction={['column', 'column', 'row']}
                 paddingInline={'1em'}
                 paddingBlock={'.5em'}
                 justifyContent={'space-between'}
               >
-                <Text
-                  mt="2"
-                  fontWeight={600}
-                  fontSize={['sm', 'md', 'lg']}
-                  width={['10em', '15em']}
-                >
-                  {name}
-                </Text>
-                <Text
-                  mt="2"
-                  fontWeight={400}
-                  fontSize={['sm', 'md', 'lg']}
-                  opacity={'.8'}
-                >
-                  {start_at}
+                <Text mt="2" fontWeight={600} fontSize={['sm', 'md', 'lg']}>
+                  {index + 1}. Event: {name}
```

Comment:
Adding _Event_ before each event name, under heading _Organized Events_ is a bit meaningless.

Reply:
True, let me update it 👍